cmd/influxd/server_integration_test.go

```diff
@@ -588,6 +588,21 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 			expected: `{"results":[{"series":[{"name":"cpu","tags":{"region":"us-east"},"columns":["time","mean"],"values":[["1970-01-01T00:00:00Z",15]]},{"name":"cpu","tags":{"region":"us-west"},"columns":["time","mean"],"values":[["1970-01-01T00:00:00Z",30]]}]}]}`,
 		},
 
+		// WHERE tag queries
+		{
+			reset: true,
```

Comment:
Added some `=` unit tests for tags. These always worked, but I wanted some tests.